gatsby-node.js

```diff
@@ -52,12 +52,14 @@ exports.onPostBuild = async function ({ graphql, reporter }, config) {
         })
 
         const index = client.index(currentIndex.indexUid)
-        await index.deleteIfExists()
+        if (index) await index.delete()
```

Comment:
```suggestion
        await index.delete()
```

Index will always exists as it does not make any http call. If the index does not exists the `task` in MeiliSearch will fail but it does not impact the following tasks.